libs/plugins/stream-client/src/lib/core/models/generate-state-from-resource.ts

```diff
@@ -25,13 +25,19 @@ export function generateStateFromResource(resource: any, schemas: ContributionSc
   const stages = (resource && resource.data && resource.data.stages) || [];
   const items = makeStageItems(stages, schemaDefs);
   const graph = makeGraphNodes(stages);
+  // ppaidi-todo : remove initialization
+  const metadata = resource.metadata || {
+    input: [],
+    output: [],
+  };
   return {
     id,
     name,
     description,
     app,
     triggers,
     handlers,
+    metadata,
```

Comment:
may be you want to add `metadata:null` to `INITIAL_STREAM_STATE` in `libs/plugins/stream-client/src/lib/core/state/stream.state.ts`